contracts/test/integration/GatewayDiamond.t.sol

```diff
@@ -896,28 +896,29 @@ contract GatewayActorDiamondTest is Test, IntegrationTestBase {
         );
     }
 
-    function testGatewayDiamond_SendCrossMessage_Fails_EoACaller() public {
-        address caller = vm.addr(100);
-        vm.startPrank(caller);
-        vm.deal(caller, DEFAULT_COLLATERAL_AMOUNT + DEFAULT_CROSS_MSG_FEE + 2);
-
-        registerSubnet(DEFAULT_COLLATERAL_AMOUNT, caller);
-
-        SubnetID memory destinationSubnet = SubnetID(0, new address[](0));
-        vm.expectRevert(abi.encodeWithSelector(InvalidXnetMessage.selector, InvalidXnetMessageReason.Sender));
-
-        gatewayDiamond.messenger().sendContractXnetMessage{value: DEFAULT_CROSS_MSG_FEE}(
-            TestUtils.newXnetCallMsg(
-                IPCAddress({
-                    subnetId: SubnetID({root: ROOTNET_CHAINID, route: new address[](0)}),
-                    rawAddress: FvmAddressHelper.from(caller)
-                }),
-                IPCAddress({subnetId: destinationSubnet, rawAddress: FvmAddressHelper.from(caller)}),
-                1,
-                0
-            )
-        );
-    }
+    // TODO: this is no longer possible because EOA cannot be subnet
```

Comment:
What is EOA? In any case - since it's not possible anymore we can remove this entirely?

Reply:
I donno who wrote this test though. If I just remove it, not sure if there are missing tests.